developer-workflow/stdlib.rst

```diff
@@ -137,3 +138,17 @@ once the authors of the module sign
 :ref:`contributor agreements <contributor_agreement>`.
 
 .. _issue tracker: https://github.com/python/cpython/issues
+
+Adding a new env var
+--------------------
+
+From Python 3.13 onwards, name environment variables with underscores for
+readability and accessibility.
```

Comment:
I initially failed to parse ["name" as a verb](https://en.wikipedia.org/wiki/Stroop_effect).  I suggest rephrasing.
```suggestion
From Python 3.13 onwards, environment variables name should use
underscores for readability and accessibility.
```

This sentence also assumes the reader is aware that the names are already uppercase, but it might be worth mentioning it explicitly:
```suggestion
Environment variables name should always be uppercase and, from Python 3.13 onwards,
use underscores for readability and accessibility.
```

Reply:
Good idea, I started off with something like yours, and tried to be clever and make it shorter but made it less clear 🙃

How about something like this?

```suggestion
Environment variables should be uppercase, and from Python 3.13 onwards, use
underscores for readability and accessibility.
```

Reply:
Technically it's the names -- not the variables themselves -- that are uppercase, but the sentence is clear even if "names" is dropped.  Dropping "always" is also ok.
In my suggestion I was going for a [parenthetical expression](https://en.wikipedia.org/wiki/Parenthesis_(rhetoric)), hence the comma after the "and".
```suggestion
Environment variables should be uppercase and, from Python 3.13 onwards,
use underscores for readability and accessibility.
```
or (with "names" added back):
```suggestion
Environment variables names should be uppercase and, from Python 3.13 onwards,
use underscores for readability and accessibility.
```
Not quite sure if a [possessive apostrophe](https://en.wikipedia.org/wiki/Apostrophe#Possessive_apostrophe) is needed here ("variables' names") -- @CAM-Gerlach?

Reply:
"Names of environment variables" is probably better